README.md

```diff
@@ -437,6 +437,75 @@ Read [Nest Docs](https://docs.nestjs.com/fundamentals/injection-scopes) for more
 
 **Dont use `I18nRequestScopeService` within controllers.** The `I18n` decorator is a much better solution.
 
+
+# Translating HttpExceptions
+
+To translate httpexceptions, we need to create a filter and translate the exception message.
+
+```typescript
+import { I18nService } from 'nestjs-i18n';
+import {
+	ExceptionFilter,
+	Catch,
+	ArgumentsHost,
+	HttpException,
+} from '@nestjs/common';
+import { Request, Response } from 'express';
+
+@Catch(HttpException)
+export class AllExceptionsFilter implements ExceptionFilter {
+	constructor(private readonly i18n: I18nService) {}
+
+	async catch(exception: HttpException, host: ArgumentsHost) {
+		const ctx = host.switchToHttp();
+		const response = ctx.getResponse<Response>();
+		const statusCode = exception.getStatus();
+
+		let message = exception.getResponse() as {
+			key: string;
+			args: Record<string, any>;
+		};
+
+		message = await this.i18n.translate(message.key, {
+			lang: host.switchToHttp().getRequest().i18nLang,
```

Comment:
`host.switchToHttp().getRequest().i18nLang` can be changed to `ctx.getRequest().i18nLang`

Reply:
sure